rafthttp/stream_test.go

```diff
@@ -59,11 +59,9 @@ func TestStreamWriterAttachOutgoingConn(t *testing.T) {
 			}
 		}
 
-		// msgc has been swapped with a new one now that prevwfc is closed
-		msgc, ok := sw.writec()
-		if !ok {
-			t.Errorf("#%d: working status = %v, want true", i, ok)
-		}
+		// msgc has either been swapped with a new one after closing prevwfc or
+		// is the initial msgc (in which case the first connection may be pending)
+		msgc, _ := sw.writec()
```

Comment:
oh, initial means the very first one. that is why i get confused. it would be great if we can make it clear.